src/ai/backend/manager/models/endpoint.py

```diff
@@ -711,6 +715,50 @@ def from_enum(cls, enum: RuntimeVariant) -> "RuntimeVariantInfo":
         return cls(name=enum.value, human_readable_name=MODEL_SERVICE_RUNTIME_PROFILES[enum].name)
 
 
+class EndpointStatistics:
+    @classmethod
+    async def batch_load_by_endpoint(
+        cls,
+        ctx: "GraphQueryContext",
+        endpoint_ids: Sequence[uuid.UUID],
+    ) -> Sequence[Optional[Mapping[str, Any]]]:
+        async def _build_pipeline(redis: Redis) -> Pipeline:
+            pipe = redis.pipeline()
+            for endpoint_id in endpoint_ids:
+                await pipe.get(f"inference.{endpoint_id}.app")
```

Comment:
Maybe I don't know the python Redis library well enough, but why is `await` used for the get operation when it's a pipelined request?

Reply:
Looks like the await is redundant here. Thanks.